great_expectations/exceptions/exceptions.py

```diff
@@ -48,11 +48,28 @@ class ExpectationSuiteError(DataContextError):
     pass
 
 
-class ExpectationSuiteNotAddedToStoreError(DataContextError):
-    def __init__(self) -> None:
+class ExpectationSuiteNotSavedError(DataContextError):
+    pass
+
+
+class ExpectationSuiteNotAddedToStoreError(ExpectationSuiteError):
+    def __init__(self, name: str) -> None:
+        super().__init__(
+            f"ExpectationSuite '{name}' must be added to the DataContext store before it can be saved. "  # noqa: E501
+            "Please call context.suites.add(), "
```

Comment:
I'd change this to `context.suites.add(<SUITE_OBJECT>)` to make the argument clear.